app/src/test/scala/org/alephium/explorer/util/SchedulerSpec.scala

```diff
@@ -83,16 +83,25 @@ class SchedulerSpec
         forAll(Gen.oneOf(zoneIds)) { zoneId =>
           //Time is 1 hour in the past so schedule happens 23 hours later
           Scheduler
-            .scheduleTime(ZonedDateTime.now(zoneId).minusHours(1).plusSeconds(2), "test-scheduler")
+            .scheduleTime(ZonedDateTime.now(zoneId).minusHours(1).plusSeconds(20), "test-scheduler")
             .toHours is 23
           //Time is 1 minute in the past so schedule happens after 23 hours (next day)
           Scheduler
             .scheduleTime(ZonedDateTime.now(zoneId).minusMinutes(1), "test-scheduler")
             .toHours is 23
           //Time is few seconds in the past so schedule happens after 23 hours (next day)
           Scheduler
-            .scheduleTime(ZonedDateTime.now(zoneId).minusSeconds(3), "test-scheduler")
+            .scheduleTime(ZonedDateTime.now(zoneId).minusSeconds(30), "test-scheduler")
             .toHours is 23
+        }
+
+        /**
+          * <a href="https://github.com/alephium/explorer-backend/issues/335">#335</a>:
+          * The following test is invalid for timezones in daylight savings.
+          *
+          * Restricting this test to run for known timezones only (Sydney & Switzerland).
+          * */
+        Seq(ZoneId.of("Australia/Sydney"), ZoneId.of("CET")) foreach { zoneId =>
```

Comment:
what will happen when Switzerland goes in light saving mode too? (+/- 1 hour)?

Reply:
Same goes with Sydney. We go in daylight savings as well (+/- 1 hour). In the comment I mentioned at least its predictable for countries we care about. This is a test-case only issue, the scheduling itself will work as expected as proven by other test-cases. The whole daylight savings and how it's computed is something I don't understand. 

If you'd like the test-case work for all daylight savings cases, I can read up on it?

Reply:
nope all good for me, thx for the explanation